content/en/tutorials/lotus-miner/cuda.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "CUDA for the Nvidia cards"
+description: "This is a step by step guide on how to configure and use CUDA architecture for Lotus instead of OpenCL for Nvidia cards."
+lead: "This is a step by step guide on how to configure and use CUDA architecture for Lotus instead of OpenCL for Nvidia cards."
+draft: false
+menu:
+    tutorials:
+        parent: "tutorials-providers"
+weight: 130
+toc: true
+---
+
+## Installing latest stable Nvidia Drivers
+
+Note that many Linux distributions provide their own packages of the NVIDIA Linux Graphics Driver in the distribution's native package management format. This may interact better with the rest of your distribution's framework, and you may want to use this rather than NVIDIA's official package.
+
+1. Download the latest stable package for your card using [here.](https://www.nvidia.com/download/index.aspx)
+
+2. Once you have downloaded the driver, change to the directory containing the driver package and install the driver by running, as root, sh ./NVIDIA-Linux-x86_64-<driver version>.run
+Eaxmple:
+
+```shell
+mkdir ~/nvidia-drivers
+cd ~/nvidia-drivers
+wget https://us.download.nvidia.com/XFree86/Linux-x86_64/510.54/NVIDIA-Linux-x86_64-510.54.run
+sudo sh NVIDIA-Linux-x86_64-510.54.run
+```
+
+3. One of the last installation steps will offer to update your X configuration file. Either accept that offer, edit your X configuration file manually so that the NVIDIA X driver will be used, or run nvidia-xconfig
+
+{{< alert icon="tip">}}
+There are some changes in latest Nvidia driver, so if you upgrade your driver remember to check nvidia-smi - it will always display the correct name for the GPU.
+
+If you are using a nvidia driver below < 460.91.03
+```shell
+export BELLMAN_CUSTOM_GPU="GeForce RTX 3090:10496"
+```
+If you are using a Nvidia driver above > 510.47.03
+```shell
+export BELLMAN_CUSTOM_GPU="NVIDIA GeForce RTX 3090:10496"
+```
+(3090 was used in this example, remember to edit it with your GPU and amount of Cuda cores)
+{{< /alert >}}
+
+## Installing the latest CUDA toolkit
+
+1. Download the latest CUDA toolkit [here.](https://developer.nvidia.com/cuda-toolkit)
+
+2. Once you have downloaded the cuda-toolkit, change to the directory containing the installer and install by running, as root, sh cuda_<version>.run
+Eaxmple:
+
+```shell
+mkdir ~/cuda
+cd ~/cuda
+wget https://developer.download.nvidia.com/compute/cuda/11.6.1/local_installers/cuda_11.6.1_510.47.03_linux.run
+sudo sh cuda_11.6.1_510.47.03_linux.run
+```
+
+2. You can use spacebar to select and deselect an installation options. You need to install only the cuda-toolkit and no other packages.
+
+3. Hit ENTER to begin the installation and wait for it to finish.
```

Comment:
1. Remove empty spaces between lines.
2. Fix typo `eaxmple`.
3. Indent codeblocks in list items.
4. Surround things like `ENTER` in code tags.